ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -715,7 +715,8 @@ Rectangle {
 
         if (messageInputField.readOnly) {
             messageInputField.readOnly = false;
-            messageInputField.cursorPosition = d.copyTextStart + QClipboardProxy.text.length;
+            messageInputField.cursorPosition = d.copyTextStart + QClipboardProxy.text.length +
+            (StatusQUtils.Emoji.hasEmoji(QClipboardProxy.html) ? 1 : 0);
```

Comment:
This will, on every keystroke, check whether clipboard has an emoji? :thinking:

Reply:
only if messageInputField.readOnly